### PR TITLE
Added navbar UI for login pages and replist, added tabs in the sidebar for tab.html, added required fields for image upload and new post

### PR DESCRIPTION
**Pull request by jindaln**

### CODE REVIEW

File: src/main/webapp/politicianPage.html [hunk 1]
```diff
@@ -3,39 +3,50 @@
...
-    <!-- Page Content -->
-    <div id="content" class="w3-main w3-padding-large" style="margin-left:40%">
+    <!-- Navbar -->
+    <div class="w3-top">
+        <div class="w3-bar w3-theme-d2 w3-left-align w3-large">
+            <a id="feed" class="w3-bar-item w3-button w3-padding-large w3-theme-d4"><i class="fa fa-home w3-margin-right"></i>Feed</a>
+        </div>
+    </div>
+    
+    <div style="margin-top:80px">
```
Comment (jynCoder) [inline src/main/webapp/politicianPage.html]: For consistency purposes with other html files, can all styling in style tags be moved to css files.

File: src/main/java/com/google/sps/data/DatastoreEntityToObjectConverter.java [hunk 2]
```diff
@@ -57,6 +57,8 @@ protected static Post convertPost(Entity postEntity)
         long timestamp = (long) (postEntity.getProperty(Constants.POST_ANSWER));
         Comment answer = convertComment(answerId);
         List<Comment> comments = convertCommentsFromPost(postEntity); 
+        System.out.println("these r the tabs");
```
Comment (epoms) [inline src/main/java/com/google/sps/data/DatastoreEntityToObjectConverter.java:60]: delete

File: src/main/java/com/google/sps/servlets/RepresentativeListServlet.java [hunk 3]
```diff
@@ -34,7 +34,7 @@
     private final String apiKey;
 
     public RepresentativeListServlet() {
-        apiKey = System.getenv(Constants.CIVIC_API_KEY);
+        apiKey = System.getenv(Constants.CIVIC_API_KEY);
```
Comment (epoms) [inline src/main/java/com/google/sps/servlets/RepresentativeListServlet.java:37]: extra space

File: src/main/java/com/google/sps/servlets/RepUploadImageServlet.java [hunk 4]
```diff
@@ -74,7 +74,7 @@ private String getBlobKeyUrl(HttpServletRequest request, String formInputElement
         if (blobKeys == null || blobKeys.isEmpty()) {
             return(Constants.DEFAULT_POLITICIAN_IMAGE_URL);
         } else {
-            return (SERVE_URL + blobKeys.get(0).getKeyString());
```
Comment (epoms) [inline src/main/java/com/google/sps/servlets/RepUploadImageServlet.java:77]: you want this space, you actually want it in the line above. I'd also move away from adding the parentheses unless its a really complex statement.

File: src/main/java/com/google/sps/servlets/NewPostServlet.java [hunk 5]
```diff
@@ -64,7 +64,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
             commentId = DatastoreManager.insertCommentInDatastoreIfNonToxic(name, comment);
         } catch(ToxicCommentException e) {
             logger.error(e);
-            throw new ServletException("Error: " + e.getMessage(), e);
+            String redirect = (feedBool == true) ? "feed.html?name=" + URLEncoder.encode(repName) :
```
Comment (v1shanker) [inline src/main/java/com/google/sps/servlets/NewPostServlet.java:67]: feedBool is sufficient

**Pull request merged**